pyfr/solvers/base/system.py

```diff
@@ -186,6 +187,9 @@ def _gen_kernels(self, eles, iint, mpiint, bcint):
     def rhs(self, t, uinbank, foutbank):
         pass
 
+    def compute_grads(self, t, uinbank):
+        raise NotImplementedError()
```

Comment:
We can improve the error message here to indicate the system in question.

Reply:
Done